lakefile.lean

```diff
@@ -5,9 +5,13 @@ open Lake DSL System
 package mathport {
   dependencies := #[{
     name := "mathlib",
-    src := Source.git "https://github.com/dselsam/mathlib4.git" "lake" none,
+    src := Source.git "https://github.com/leanprover-community/mathlib4.git" "master" none,
```

Comment:
This may not always work. For example, https://github.com/leanprover-community/mathlib4/commit/6b8b2390f3a404c446e9d9d80ea20fa67448d2a4 breaks the mathport build.